src/gui/PlayClass.cpp

```diff
@@ -95,7 +95,7 @@
 #endif
 
 PlayClass::PlayClass() :
-	demuxThr(NULL), vThr(NULL), aThr(NULL),
+	demuxThr(NULL), vThr(NULL), aThr(NULL), aRatioName("auto"),
```

Comment:
Not necessary to moving only one variable here. Also moving everything here won't look fine (I think)...